src/constants/signals.js

```diff
@@ -12,3 +12,19 @@ export const SIGNAL_EXPAND = {
   CRITICAL_ONLY: 2,
   ALL: 4,
 };
+
+export const ALERT_SEVERITY = {
+  CRITICAL: 'CRITICAL',
+  NOT_ALERTING: 'NOT_ALERTING',
+  NOT_CONFIGURED: 'NOT_CONFIGURED',
+  WARNING: 'WARNING',
+};
+
+export const SIGNAL_TOOLTIP = {
```

Comment:
This should be in `src/constants/ui-content.js` since it's actual content that shows up in the UI.